smell-test

```diff
@@ -170,9 +174,9 @@ def select_DNS(pkt):
     except Exception as e:
         print(e)
 
-cache_path()
 print ('[**] Beginning "Smell Test"')
 try:
     sniff(iface=interface, filter=filter_bpf, store=0,  prn=select_DNS)
 except OSError as e:
+    # note: this works on Linux but OS X segfaults when the interface is wrong lmao
```

Comment:
Hmmm doesn't that segfault happen along with a warning about [`netifaces`](https://pypi.python.org/pypi/netifaces)? Maybe installing it would help? I dunno.

Reply:
Probably. I'll look into it at some point